src/actions/index.js

```diff
@@ -12,26 +12,28 @@ export function fetchSearchIndexSuccess(searchIndex) {
     type: actionTypes.FETCH_SEARCH_INDEX_SUCCESS,
     searchIndex,
   };
-}
+} 
```

Comment:
Try not to leave extra spaces at the end of lines. Linter might actually catch this if you run lint.

Reply:
I will make sure to run the linter in the future!